cst438week1_assignment1/src/main/java/cst438/Movie.java

```diff
@@ -27,18 +25,16 @@
 		
 		@NotNull
 		@Max(100)
+		@Min(0)
 		private float rating;
 		
-
-		
+		//Default constructor includes the new date in string format.
 		public Movie() {
 			java.util.Date dt = new java.util.Date();
 			java.text.SimpleDateFormat sdf = new java.text.SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
 			this.date = sdf.format(dt);
 		}
-
-
-
+		
 		public Movie(long id, @NotNull @Size(min = 3, max = 25) String title,
 				@NotNull @Size(min = 3, max = 25) float rating) {
```

Comment:
```suggestion
				@NotNull float rating) {
```
I believe you were asking for a Size of 3-25 characters but since it is just a rating, it will only be one int.